conf/application.conf

```diff
@@ -25,24 +25,6 @@ play.http.errorHandler = "handlers.ErrorHandler"
 
 location.canonical.list = "location-autocomplete-canonical-list.json"
 
-play.http.filters = play.api.http.EnabledFilters
-
-play.filters.enabled = [
```

Comment:
CLARIFICATION: Remind me, how the filters are configured now?

Reply:
I'm hoping that the magic happens in bootstrap - according to these instructions - https://github.com/hmrc/bootstrap-play and the `include "frontend.conf"` at the top of this file.